src/timeline/route.rs

```diff
@@ -109,5 +113,34 @@ pub fn render_timeline_route(
                 None
             }
         }
+
+        TimelineRoute::Quote(id) => {
+            let txn = if let Ok(txn) = Transaction::new(ndb) {
+                txn
+            } else {
+                ui.label("Quote of unknown note");
+                return None;
+            };
```

Comment:
this can just be: `let txn = Transaction::new(ndb).expect("txn")`

so we can catch double transactions at runtime